config.example/group_vars/all.yml

```diff
@@ -179,6 +179,17 @@ maas_single_node_install: true
 
 maas_kvm: false
 
+################################################################################
+# SOFTWARE MODULES (SM)                                                        #
+################################################################################
+sm_prefix: /sw                                 # has to be in a shared location
+sm_module_root: "{{ sm_prefix }}/modules"
+sm_module_path: "{{ sm_module_root }}/all"
+sm_software_path: "{{ sm_prefix }}/software"
+sm_files_path: "{{ sm_prefix }}/easybuild_files"
+# need to put this in github/DeepOps
```

Comment:
We should do this before this PR is merged, can you file a PR against: https://github.com/DeepOps/easybuild_files

Reply:
can you add me to that one, or push something so that I can issue a pull request

Reply:
done.